.gitignore

```diff
@@ -5,10 +5,14 @@
 /customCodes.json
 /userOnlineTime.txt
 /data/
-/settings/
+/settings/*
 
-# Secrets
+# Don't ignore the example settings file
+!/settings/settings.example.json
```

Comment:
We should add this to `.prettierignore` as well.

Reply:
Also done!